src/Symfony/Component/Finder/Tests/Iterator/VcsIgnoredFilterIteratorTest.php

```diff
@@ -74,6 +74,24 @@ public static function getAcceptData(): iterable
             ],
         ];
 
+        yield 'simple file - .gitignore and in() from repository root' => [
```

Comment:
Maybe I'm missing something but I don't think this test completely covers the issue you're fixing. Shouldn't we have a Git repository nested in another one, both with a `.gitignore` file, showing that finding files from the nested repository doens't take the parent `.gitignore` file into account?

Reply:
I added a second test to explicitly test for the nesting now.